euler_support/urdf/euler.xacro

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" ?>
 <robot name="euler" xmlns:xacro="http://ros.org/wiki/xacro">
-        <xacro:include filename="$(find euler_support)/urdf/euler_macro.xacro" />
-        <xacro:euler prefix=""/>
+    <xacro:include filename="$(find euler_support)/urdf/euler_macro.xacro" />
+    <xacro:euler prefix="euler_"/>
```

Comment:
Try to setup a xacro argument for the prefix [(See section 6)](http://wiki.ros.org/xacro) that defaults to an empty string so that we can set its value from either a launch file or the command line.

Reply:
On a related note, the tf tree of the whole robot ,with the exception of the vacuum gripper, should look as shown in the picture.
![euler_original_tf_graph](https://user-images.githubusercontent.com/3279718/39940265-5177a28c-551e-11e8-922d-ca819467b559.png)
You can view the **tf_tree**  by running 
```
rosrun rqt_tf_tree rqt_tf_tree
```